docs/source/deployment/deploy-model-locally.rst

```diff
@@ -77,7 +77,60 @@ CSV input must be a valid pandas.DataFrame CSV representation. For example:
 JSON Input
 **********
 
-JSON input must be a dictionary with exactly one of the following fields, further specifying the input data's type and encoding:
+You can either pass a flat dictionary corresponding to the desired model payload or wrap the
+payload in a dict with a dict key that specifies your payload format. 
+
+Raw Payload Dict
+^^^^^^^^^^^^^^^^
+
+If your payload is in a format that your mlflow served model will accept and it's in the supported
+models below, you can pass a raw payload dict.
+
+.. list-table::
+    :widths: 20 40 40
+    :header-rows: 1
+    :class: wrap-table
+
+    * - Supported Model Type
+      - Description
+      - Example
+    * - OpenAI Chat
+      - `OpenAI chat request payload <https://platform.openai.com/docs/api-reference/chat/create>`_.
+        Note that openai's ``model`` kwarg should not be included because we are querying the
+        mlflow served model. However, any other openai.chat parameter defined in our served model's 
+        signature will be applied.
+      - 
+        .. code-block:: python
+
+          {"messages": [{"role": "user", "content": "Tell a joke!"}], "temperature": 0.0}
+
+.. code-block:: python
```

Comment:
```suggestion
    * - OpenAI Chat
      - `OpenAI chat request payload <https://platform.openai.com/docs/api-reference/chat/create>`_.†
      - 
        .. code-block:: python

          {"messages": [{"role": "user", "content": "Tell a joke!"}], "temperature": 0.0}

† Note that the ``model`` argument **should not** be included when using the OpenAI APIs, due to its configuration being set by the MLflow model instance. All other parameters can be freely used, provided that they are defined within the ``params`` argument within the logged model signature.

.. code-block:: python
```

I think the table might look a bit cleaner if we add this note as a local footnote. WDYT?

Reply:
Looks great